spec/spec_helper.rb

```diff
@@ -1 +1,23 @@
+if ENV['COVERAGE']
```

Comment:
This load are not there "just for simplecov", they are there because if where are not loading them, then we can not have a complete coverage analysis. This is an option to have in the specs configuration, this is not an option to have in a rake task. 

I agree, this should be wrapped in a helper that basically load everything, but still an method to be call in here. At less for my understanding.